axelrod/strategies/shortmem.py

```diff
@@ -0,0 +1,45 @@
+from axelrod import Actions, Player
+from axelrod.actions import Action
+
+C, D = Actions.C, Actions.D
+
+class ShortMem(Player):
+    """
+    A player starts by always cooperating for the first 10 moves.
+    
+    The opponent answers are stored in the memory, whose maximum size is 10 results. From the tenth round on, the program analyzes the memory, and compare the number of defects and cooperates of the opponent, based in percentage. If cooperation occurs 30% more than defection, it will cooperate.
```

Comment:
This all looks good to me, could you keep the number of characters on each line to less than 80 please? So it just needs this docstring to line break a bit earlier.